### PR TITLE
Quick fixes

### PR DESCRIPTION
**Pull request by GenericNerd**
- Corrected aircraft ID being given as a string in phpVMS 5 handler
- Added required `route` POST field to /flight/complete endpoint

### CODE REVIEW

File: openapi.json [hunk 1]
```diff
@@ -709,6 +710,13 @@
                                         "format": "float",
                                         "minimum": 0
                                     },
+                                    "route": {
+                                        "type": "array",
+                                        "items": {
+                                            "type": "string",
+                                            "pattern": "^((0?[1-9]|[1-2]\\d|3[0-6])[LCR]?)|([A-Z]{5})|([A-Z]{3})|([A-Z]{1-3})$"
```
Comment (jordanhavard) [inline openapi.json:717]: Not all routes will conform to this regex. Does the application also have this regex or is this just API related (and VAs can change this?) @GenericNerd

**Review by jordanhavard — COMMENTED** [review]
Question about route regex

File: openapi.json (hunk 1 above)
Comment (GenericNerd) [reply openapi.json:717]: This is only for the API specification to be able to generate routing that is loosely correct. The actual API code only expects an array of strings in this case.
